package.json

```
@@ -10,6 +10,10 @@
     "index.js",
     "index.mjs"
   ],
+  "exports": {
+    "import": "./index.mjs",
+    "require": "./index.js"
+  },
```

I'd suggest the longer form:
```suggestion
  "exports": {
    ".": [
      {
        "import": "./index.mjs",
        "require": "./index.js",
        "default": "./index.js"
      },
      "./index.js",
    ]
  },
```
the array fallback is for node 13.0 and 13.1, and by using the object with a dot, you've reduced future diff churn if you ever add a new entry point, like `./package.json` for example.

---

Note as well, if you wanted this to be a minor, you could add `./index`, `./index.js`, `./package`, and `./package.json`, and i think that'd be exhaustive?

---

Technically I guess it could still be argued that it's major if you had a require extension helper to allow you to require markdown files.

---

I'll release this as breaking so it's a clean, simple break. It's not like it's a big enough module for it to matter much if people end up with two versions of it for a while.

---

I think it’s reasonable not to support any no stance se extensions.

Breaking is fine too :-)